misc/formatter/modules/formatter-core/src/test/resources/statements/call/source/call_statement_3.bal

```diff
@@ -0,0 +1,21 @@
+function calculate() returns int {
+    int result = subtract(add([21, 45, 6, 12, 67, 89], [1,5,6,9]),
```

Comment:
Shall we also add samples like in the issue where we have methods calls in a `return` statement? Also add tests with multiple arguments eg: 4 args in 4 lines